app/react/settings/people/PeopleToolbarContainer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Link from "../../react-transition/Link";
+
+const PeopleToolbarContainer = () => (
+    <div className="toolbar">
+        <div className="fab">
+            <Link
```

Comment:
We should be able to user react router Link here, sine we're navigating to a react component.